### PR TITLE
layout of list QR Codes

### PR DESCRIPTION
**Pull request by ghost**
## Description
Layout list QR codes not print yet 
## Reference Requirements
- Product Backlog ID: PBI-9
- Task ID: #9.1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
No

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
No

### CODE REVIEW

File: app/Http/Controllers/Admin/QrcodeController.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class QrcodeController extends Controller
```
Comment (at-luanle) [inline app/Http/Controllers/Admin/QrcodeController.php]: QrCodeController , em coi sua lai cho dep code

File: resources/lang/en/qrcode.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
+return [
+    'title'         => 'List QR Codes',
```
Comment (at-luanle) [inline resources/lang/en/qrcode.php]: QR Code em nen viet sat lai : QrCodes

File: resources/lang/en/qrcode.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    'admin'         => 'Admin',
+    'qrcodes'       => 'QR Codes',
```
Comment (at-luanle) [inline resources/lang/en/qrcode.php]: day cung vay

File: app/Http/Controllers/Admin/QrcodeController.php (hunk 1 above)
Comment (at-nghiavo) [reply app/Http/Controllers/Admin/QrcodeController.php]: không cần đâu anh, em có tham khảo trước

**Comment by at-luanle** [comment]
LGTM

**Review by ghost — APPROVED** [review]

**Pull request merged**